tests/Doctrine/Tests/Common/Cache/RedisCacheTest.php

```diff
@@ -35,18 +35,6 @@ public function testGetRedisReturnsInstanceOfRedis()
         $this->assertInstanceOf('Redis', $this->_getCacheDriver()->getRedis());
     }
 
-    public function testSerializerOptionWithOutIgbinaryExtension()
```

Comment:
Try something, then we review? :-P

In general, yes, a new symbol is a better solution than a BC break, especially on such a widespread library.